### ISSUE TITLE
Text spacing adjustment doesn't work on buttons

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#90]

**Comment by gabalafou** [comment]
I think this is actually just an issue with Accessibility Insights and doesn't have anything to do with PST.

Steps I followed:

1. Visited [Technique C36: Allowing for text spacing override](https://www.w3.org/WAI/WCAG21/Techniques/css/C36)
2. Added the [text spacing bookmarklet](https://www.html5accessibility.com/tests/tsbookmarklet.html) to my browser
3. Visited the [PyData Sphinx Theme home page](https://pydata-sphinx-theme.readthedocs.io/)
4. Clicked the bookmarklet
5. Observed that the text spacing increased everywhere, including in the buttons, where it previously did not with the Accessibility Insights visual helper

In the following screenshot, notice that the text spacing has increased in the version switcher opener button, the open version switcher menu, the "more" dropdown button, and the search opener button:

![](https://github.com/user-attachments/assets/d489331a-7434-46f2-8406-c9971be284d9)

I also tested the home page in Polypane and got the same conforming result. 

Therefore, I believe there's nothing to do with this issue, and so I am closing it.
